katy-nike-collection.php

```diff
@@ -2,10 +2,58 @@
 /**
  * retrieves a DB connection to the collection of shoes DB
  */
-$db = new PDO('mysql:host=db; dbname=katy-collection', 'root', 'password');
-$db->setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC);
-return $db;
-    
+
+function getDb() {
+    $db = new PDO('mysql:host=db; dbname=katy-collection', 'root', 'password');
+    $db->setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC);
+    return $db;
+}
+
+
+function getSneakers($db)
+{
+    $query = $db->prepare("SELECT `name`, `year-released`, `color-way`, `size`, `price`, `images` FROM `sneakers`;");
+    $query->execute();
+    return $query->fetchAll();
+}
+
+$db = getDb();
+$result = getSneakers($db);
+
     echo '<pre>';
 var_dump($result);
```

Comment:
For some reason the £ symbol does not display properly.  I tried to find the cause of the problem yesterday but I had no luck